virttest/utils_test/libvirt.py

```diff
@@ -3896,6 +3896,7 @@ def check_cmd_output(cmd, content, err_ignore=False, session=None):
     s, cmd_output = utils_misc.cmd_status_output(cmd, shell=True,
                                                  ignore_status=err_ignore, session=session)
     pattern_list = [content] if not isinstance(content, list) else content
+    LOG.debug('Ran command "%s" with output "%s"' % cmd, cmd_output)
```

Comment:
utils_misc.cmd_status_output(verbose=True) already supports to log the message your want when no session, so if you want to enhance this, it is better to add logging in utils_misc.cmd_status_output() when there is a  session passed in.

For the `LOG.debug(),` we need to pay attention to the usage of ',' instead of '%'. So it should be 
`LOG.debug('Ran command "%s" with output "%s"',  % (cmd, cmd_output))`

Reply:
Sorry, I made a mistake in above comment for LOG.debug.
 It should be `LOG.debug('Ran command "%s" with output "%s"', cmd, cmd_output)`, right?

Reply:
Fixed.